Added Release Notes 6.4.0 (#73)
owncloud/docs-main: modules/ROOT/pages/ocis_release_notes.adoc

[discrete]
=== Enhanced Accessibility for the Datepicker UI

The Datepicker UI has been replaced with the browser’s native datepicker. This change improves accessibility by offering a more familiar and consistent experience across different devices and browsers. Note: We treat any accessibility violation as a bug and kindly ask you to contact us if you encounter any digital barriers.

[discrete]
== Tiles View: Lazy Loading and Select All Checkbox

Tiles View now supports lazy loading with an animation for smoother performance, and includes a “Select All” checkbox for easier bulk actions, similar to List View.

[discrete]
== Open Cloud Mesh - Bugfixes

The Open Cloud Mesh (OCM) protocol, which supports cross-cloud file sharing, has been improved with several bug fixes. These updates enhance stability and reliability, reducing potential issues when federating between different clouds. This feature remains experimental which means that it may break and individual production usage must be agreed with ownCloud Support.

[discrete]
=== Known Issues

> * Open Cloud Mesh *
> 
> The Open Cloud Mesh (OCM) integration has several known issues, including federated shares not syncing with desktop and mobile clients, errors when hiding shares, and issues with the “Disable Sync” function. Users should be aware of these limitations while using the experimental feature. For a complete list of known issues, visit the https://github.com/owncloud/ocis/issues/9735[GitHub issue tracker].

Reviewer: ```suggestion
*Open Cloud Mesh*

The Open Cloud Mesh (OCM) integration is an experimental feature and has several known issues, including federated shares not syncing with desktop and mobile clients, errors when hiding shares, and issues with the “Disable Sync” function. Users should be aware of these limitations. For a complete list of known issues, visit the https://github.com/owncloud/ocis/issues/9735[GitHub issue tracker].
```
